src/communication/communication_server_channel.hpp

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (C) 2008 Search Solution Corporation. All rights reserved by Search Solution.
+ *
+ *   This program is free software; you can redistribute it and/or modify
+ *   it under the terms of the GNU General Public License as published by
+ *   the Free Software Foundation; either version 2 of the License, or
+ *   (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program; if not, write to the Free Software
+ *  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
+ *
+ */
+
+/*
+ * communication_server_channel.hpp
+ */
+
+#ifndef _COMMUNICATION_SERVER_CHANNEL_HPP_
+#define _COMMUNICATION_SERVER_CHANNEL_HPP_
+
+#include "communication_channel.hpp"
+
+#include <string>
+
+namespace cubcomm
+{
+  enum class server_server //server to server commands
+  {
+    CONNECT_TRANSACTION_SERVER, // transaction to page server
+    CONNECT_SLAVE_CONTROL, //slave server to a control channel
```

Comment:
Remove channel type specific to HA redesign
```suggestion
    CONNECT_TRANSACTION_SERVER, // transaction to page server
```

Reply:
done